### PR TITLE
Clarify deprecation of Email constraint without egulias/email-validator

### PR DESCRIPTION
**Pull request by rpkamp**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Review by ro0NL — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Comment by xabbuh** [comment]
I think we should update the `UPGRADE-5.0.md` file and the deprecation messages in the PHP code (in the constraint and the validator) too.

**Comment by rpkamp** [comment]
> I think we should update the `UPGRADE-5.0.md` file and the deprecation messages in the PHP code (in the constraint and the validator) too.

Good point. I'll do that somewhere later today :+1:

**Review by ogizanagi — APPROVED** [review]

**Comment by rpkamp** [comment]
`UPGRADE-5.0.md` and `Email` constraint updated.

File: src/Symfony/Component/Validator/Constraints/Email.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public function __construct($options = null)
 
         if ((self::VALIDATION_MODE_STRICT === $this->mode || true === $this->strict) && !class_exists(StrictEmailValidator::class)) {
             // throw new LogicException(sprintf('The "egulias/email-validator" component is required to use the "%s" constraint.', __CLASS__));
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Constraints/Email.php]: makes sense here as well i guess :)

Comment (rpkamp) [reply src/Symfony/Component/Validator/Constraints/Email.php]: Indeed! I'll do that in a bit :slightly_smiling_face:

Comment (rpkamp) [reply src/Symfony/Component/Validator/Constraints/Email.php]: Done! :slightly_smiling_face:

**Review by xabbuh — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Pull request merged**